utils/vfs/vfs.cpp

```diff
@@ -0,0 +1,123 @@
+#include <algorithm>
+#include <fuzzuf/utils/filesystem.hpp>
+#include <fuzzuf/utils/vfs/vfs.hpp>
+#include <fuzzuf/exceptions.hpp>
+namespace fuzzuf::utils::vfs {
+
+VFS::VFS(
+  std::vector< fs::path > &&allowed_path_
+) {
+  if( !allowed_path_.empty() ) {
```

Comment:
Probably we can make this function less nested if it processes `if( allowed_path_.empty() ) {` first.

Reply:
fixed